api/src/main/java/io/minio/MakeBucketArgs.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.minio;
+
+public final class MakeBucketArgs {
```

Comment:
Please note that this class expect `objectLock` is required to each and every `<API>Args` class in future. We do not copy/paste this class, instead we need a base class `BucketArgs` and other classes inherit it.

APIs accepting object name as argument require different style and we will be fixing the design in one of such PR.

@kannappanr Any intermediate release in the middle will break backward compatibility. Its required to have close attentions on these PRs.